### DISCUSSION TITLE
Sui Move build  & Sui Move publish 拉取git包改进

### DISCUSSION THREAD

**Discussion by uvd** [move-cn/letsmove#1199] [Ideas]
1. 如果没有安装git   提示安装git或者指定git安装路径
2. 如果网络连接失败提示网络链接失败
3. 正常的话提示git获取的代码进度和拉取日志，防止网速慢的人以为卡了

代码合并奖励100SUI

**Comment by Azleal** [comment]
具体执行什么命令的时候出现这个问题？

**Reply by AikoTsumugi** [reply]
+1 求解释具体场景

**Comment by n-WN** [comment]
分配认领

**Comment by foxmoon** [comment]
只要git clone就算完成？
